<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFzusWay2Widget</class>
 <widget class="QWidget" name="CFzusWay2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{border-image: url(:/div/fzus/alu2.png);}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="mRg1WrVal">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1OutVal">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>528</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2Val">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>109</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1IncVal">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1WrHint">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>74</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3RstVal">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>373</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Out3Hint">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>152</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1OutHint">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>522</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1OutVal">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>313</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">000000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3WrHint">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>386</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3WrVal">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>393</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mDVal">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>5</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3RstHint">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>367</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2OutHint">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>522</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2Val">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>554</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Out3Val">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>164</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1WrHint">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>72</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Val">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>107</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1Val">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>554</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3Val">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>408</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">000000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1WrVal">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>79</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1IncHint">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>94</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Val">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>106</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2OutVal">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>528</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mXor1Val">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>215</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">000000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mXor1NotVal">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mXor1NotHint">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>184</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3ShlVal">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>412</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3ShlHint">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>405</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4Val">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>472</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4ShlVal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>442</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4ShlHint">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>435</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1CHint">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>236</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1CVal">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>243</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2WrVal">
    <property name="geometry">
     <rect>
      <x>756</x>
      <y>82</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2WrHint">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>75</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2ShlHint">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>106</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2ShlVal">
    <property name="geometry">
     <rect>
      <x>756</x>
      <y>113</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3Out8Val">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>486</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3Out8Hint">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>474</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
